Podcast/Episode.swift

```diff
@@ -128,14 +108,15 @@ class Episode: NSObject {
         return dateFormatter.string(from: dateCreated)
     }
     
-    func attributedDescriptionString() -> NSAttributedString {
+    func attributedDescriptionString() -> NSMutableAttributedString {
```

Comment:
Why return a mutable one? Just use a mutable one and return a regular one? Idk if this is actually more efficient but it doesn't seem like it should be mutable after being returned.